GOVERNANCE.md

```diff
@@ -0,0 +1,139 @@
+# Governance
+
+This document defines the governance process for the GitOps Working Group and community and OpenGitOps project.
+
+## Roles
+
+### All Members
+
+This is an open public working group welcoming anyone who would like to join and help.
+There are no special requirements for membership apart from the responsibilities listed  below.
+Members are not required to have any explicit roles or additional responsibilities, but they may also have formally assigned roles (see below).
+See the Working Group [README](README.md) for information about volunteering for committees and projects.
+Note that voting requires additional conditions (see [Elections](#elections)).
+
+Responsibilities:
+
+- Must follow the [Code of Conduct](CODE_OF_CONDUCT.md)
+- May not create the impression that they have any authority or formal responsibilities within the GitOps WG other than their explicitly assigned roles
+
+### Committees
+
+Working Group [Members](#all-members) may volunteer to join WG Committees and help with specific projects according to their interest and ability.
+Responsibilities vary per committee, but in general please remember everyone is a volunteer.
+Committees follow processes to allow for progress so long as they maintain inclusiveness, transparency, and respect for all involved.
+
+Current WG committees:
+
+- Principles Committee
+  - Responsible for drafting the revised GitOps Principles (see [Timeline](#timeline))
+- Events Committee
+  - Responsible for organizing events such as GitOpsCon, and supporting other events in the wider GitOps community (see [Timeline](#timeline))
+- Communications Committee
+  - Reponsible for coordinating public communications on behalf of the working group
+
+### Maintainers
+
+Working Group and OpenGitOps project Maintainers are [Members](#all-members) who have shown significant and sustained contributions in the GitOps WG.
+The initial set of Maintainers were drawn from the organizations that proposed the creation of the working group.
+Current Maintainers are listed in a [MAINTAINERS](./MAINTAINERS) file at the root of this git repository.
+
+Responsibilities:
+
+- Enable and promote GitOps Working Group community values
+- Engage with the wider GitOps community through appropriate [communication channels](./README.md#how-to-get-involved)
```

Comment:
Update link to the new location of GitOps WG community involvement info.

To-do: Update https://github.com/open-gitops/.github/blob/main/CONTRIBUTING.md as a separate PR in the community health files repo. The plan is to move the appropriate steps directly into this document. For now link to what we have. 

```suggestion
- Engage with the wider GitOps community through appropriate [communication channels](https://github.com/cncf/tag-app-delivery/blob/main/gitops-wg/README.md#community)
```